proto/v1alpha1/rode.proto

```diff
@@ -192,3 +201,8 @@ message RegisterCollectorRequest {
 message RegisterCollectorResponse {
   map<string, grafeas.v1beta1.Note> notes = 1;
 }
+
+message CreateNoteRequest {
+  string note_id = 1 [(google.api.field_behavior) = REQUIRED];
+  grafeas.v1beta1.Note note = 2 [(google.api.field_behavior) = REQUIRED];
```

Comment:
didn't realize that we could specify arguments as required in the proto, or perhaps that's only for the grpc gateway code.  but I saw this in grafeas protos and decided to use it as well.

Reply:
It may just be for the gateway. I know required/optional were available in proto2, but dropped from proto3.